Resizetizer.NT/Resizer.cs

```diff
@@ -38,33 +35,34 @@ public static string GetFileDestination(SharedImageInfo info, DpiPath dpi, strin
 			return destination;
 		}
 
-		public static ResizedImageInfo CopyFile(SharedImageInfo info, DpiPath dpi, string intermediateOutputPath, string inputsFile, ILogger logger, bool isAndroid = false)
+		public ResizedImageInfo CopyFile(DpiPath dpi, string inputsFile, bool isAndroid = false)
 		{
-			var destination = Resizer.GetFileDestination(info, dpi, intermediateOutputPath);
+			var destination = GetFileDestination(dpi);
 			var androidVector = false;
 
-			if (isAndroid && info.IsVector && !info.Resize)
+			if (isAndroid && Info.IsVector && !Info.Resize)
 			{
 				// Update destination to be .xml file
-				destination = Path.ChangeExtension(info.Filename, ".xml");
+				destination = Path.ChangeExtension(destination, ".xml");
```

Comment:
This _may_ have been a bug... When processing a non-resized, android vector, it would actually create the file in the source directory. For example:

Assume file in project: `my_image.svg`
This would make a copy right next to it as `my_image.xml`

This should have been in the intermediate directory.

Reply:
Sounds like a bug, this code path is not super useful anyway since the conversion doesn't support a lot of svg concepts to translate...